core/lib/multivm/src/versions/vm_1_3_2/utils.rs

```diff
@@ -222,10 +224,10 @@ pub fn create_test_block_params() -> (BlockContext, BlockProperties) {
 }
 
 pub fn read_bootloader_test_code(test: &str) -> Vec<u8> {
```

Comment:
This function can be removed; ditto for similar functions in other VM versions older than `vm_latest`. They were previously used in unit tests, but now they are unused.